### DISCUSSION TITLE
Firefly just works with PHP 8.3 ?

### DISCUSSION THREAD

**Discussion by lucaslgr** [firefly-iii/firefly-iii#8784] [General]
```bash
root@DietPi:/var/www/firefly-iii# php artisan firefly-iii:upgrade-database
Composer detected issues in your platform:

Your Composer dependencies require a PHP version ">= 8.3.0". You are running 8.2.7.

PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.3.0". You are running 8.2.7. in /var/www/firefly-iii/vendor/composer/platform_check.php on line 22
```

I'm installing firefly in a local server that already has another applications running with PHP 8.2, so I was trying to run FireFly with the same version but it isn't working. I know that in Firefly's documentation is saying that we need the 8.3 version, however, as it is just a minor upgrade version, without any breaking changes, I was thinking that it would work fine. Am I completely wrong?

**Comment by JC5** [comment]
I'm afraid so. Firefly III only supports PHP 8.3.
